lib/inspec/runner.rb

```diff
@@ -182,58 +182,10 @@ def register_rules(ctx)
 
     private
 
-    def block_source_info(block)
```

Comment:
be aware that `inspec json` should return the same information. Therefore I disagree with the move to the rspec formatter